test/java/org/opendatakit/briefcase/reused/http/RequestBuilderTest.java

```diff
@@ -86,4 +89,11 @@ public void can_resolve_paths() {
         is(url("http://foo.com/bar/baz"))
     );
   }
+
+  @Test
+  public void can_resolve_body(){
```

Comment:
This test doesn't belong to this test class because what we're verifying is that the `RequestSpy` class can read a request body. You don't need a `Request` object for that.